### DISCUSSION TITLE
Home Assistant Automation When PC Wakes Up

### DISCUSSION THREAD

**Discussion by Kombuchelada** [timmo001/system-bridge#3244] [Help]
I'm sure this is possible, but I'm struggling to come up with a way to trigger a home assistant automation when my PC goes from "asleep" to "awake" or from "off" to "on".

I'm new to System Bridge so I'm hoping for suggestions on how to trigger this automation. Thanks!

**Comment by dpiccinati** [comment]
I'm looking for the same type of automation...also super new but I have to believe there is an answer to this.

**Comment by OlwinFroon** [comment]
I'm new as well and I'd love to have that, too.

However, I don't see a sensor suitable for that need and so I guess there might be a workaround: if one of the essential sensors (like free memory) goes to "unavailable", your PC is down ...

**Comment by dpiccinati** [comment]
I tried using some other sensors in that way. Unfortunately, it's just not very accurate and often takes a while to update. My use case was to turn on my overhead office light whenever I turned my computer on (as I don't need it otherwise) however it is just less work and more reliable to just flick the switch.

Does anyone else have a more reliable and lower latency solution for that use case by chance?

**Comment by OlwinFroon** [comment]
Well, the app send the PC's status to HA. For that, the PC needs to be up and online. 

The app usually doesn't start before your OS is up, so there's that latency and it's kinda unavoidable.

I have a similar situation (fire up the printer as soon as the PC goes online, power it down a while after the PC goes offline) and I solved it with a [Ping sensor](https://www.home-assistant.io/integrations/ping/). 
Turns out the network interface goes live a LOT sooner than Windows. 
This might work for you as well.
